index.html

```diff
@@ -33,19 +33,134 @@ <h2>I’am a Software Developer</h2>
                 <a href="#"><img src="./images/Vector16.png"></a>
                 <a href="#"><img src="./images/Vector5.svg"></a>
             </ul>
-
-            <!--<div class="image">
-                <img src="./images/illustration-header-mobile.png" alt="bg">
-            </div>-->
-
         </div>
         <div class="grid-container">
             <div class="grid-item1">
                 <h1>My Recent Works</h1>
+                <hr>
+                <img src="./images/Img Placeholder.png" alt="project1">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="list">
+                    <li class="list1"><a href="#">css</a></li>
+                    <li class="list1"><a href="#">html</a></li>
+                    <li class="list1"><a href="#">bootstrap</a></li>
+                    <li class="list1"><a href="#">Ruby</a></li>
+                </ul>
+                <div class="project"><a href="#">see project</a></div>
```

Comment:
- [ ] Again you did well adding the hover effects to this button ✔, but the `font-type` and `color` you are using here is not consistent with the overall design 🤔. Please fix this 👇🏽

### Figma design:
![image](https://user-images.githubusercontent.com/43172164/205433052-d165effe-72ae-44f4-a9e7-d092ff8af80b.png)
### Your design:
![image](https://user-images.githubusercontent.com/43172164/205433062-36d074df-a810-4c18-9b77-9f8ac154f0be.png)